Source/Private/Find-LintRemarks.ps1

```diff
@@ -6,7 +6,7 @@ function Find-LintRemarks {
         $LintLines
     )
     if ($null -eq $LintLines) {
-        return @()
+        return , @()
```

Comment:
But why?
If this is to force array then this is not the correct way of doing it. Because you would have an array of size 2.

Reply:
Yes, that is the reason. This is the only way I have yet discovered that allows you to return an empty array from a function. This is due to the way output collection is implemented in Powershell. There are links to a decent explanation in the accepted answer here: 

https://stackoverflow.com/questions/18476634/powershell-doesnt-return-an-empty-array-as-an-array

I have added an extra test that tries to make it clearer what I expect in the return value.